gdal/makefile.vc

```diff
@@ -60,16 +60,30 @@ DEFAULT_TARGETS =
 
 default:	$(TARGET_LIB) $(PLUGIN_TARGET) apps_dir
 
-staticlib:      $(LIB_DEPENDS)
-	if exist gdal.lib del gdal.lib
+CLEAN_MAIN_BUILD_OUTPUT_CMDS = \
+@SETLOCAL EnableExtensions^
+if exist gdal.lib del gdal.lib^
+if exist $(GDAL_DLL) del $(GDAL_DLL)^
+if exist $(GDAL_DLL).manifest del $(GDAL_DLL).manifest^
+if exist gdal_i.lib del gdal_i.*^
+if exist $(GDAL_PDB) del $(GDAL_PDB)^
+if exist *.ilk del *.ilk
+
+staticlib:	$(LIB_DEPENDS)
+	call <<clean_main_build_output.bat
+$(CLEAN_MAIN_BUILD_OUTPUT_CMDS)
+<<
```

Comment:
This is the cleanest way I could figure out to reuse these commands without making a target that would presumably cause the other targets to always be out of date.

Reply:
could you incorporate that as a comment in the file ?

Reply:
Absolutely!